pkgs/racket-doc/scribblings/guide/places.scrbl

```diff
@@ -55,7 +55,7 @@ then calls @racket[any-double?] on the list and binds the result to
 In DrRacket, after saving and running the above program, evaluate
 @racket[(main)] in the interactions window to create the new
 place. @margin-note*{When using @tech{places} inside DrRacket, the
-module containg place code must be saved to a file before it will
```

Comment:
There are a couple of "containg" in the Racket repo, though the other occurrences are in comments / text file.